### PR TITLE
ImagePipeline processors

### PR DESCRIPTION
**Pull request by vendruscolo**
This PR implements the feature discussed in #269

### CODE REVIEW

**Comment by kean** [comment]
Hi, @vendruscolo. This looks good! I will appreciate if you could also update [the documentation](https://github.com/kean/Nuke#default-image-pipeline) and also add a couple of unit tests.

**Comment by vendruscolo** [comment]
I've added a few tests and also updated the documentation.
Tests have been added in a new file, as I felt those three tests were a bit out of scope if done in the `ImagePipelineTests` class.
I've also created a new subsection for the documentation.

Let me know if everything looks good to you.

**Comment by kean** [comment]
Thanks, @vendruscolo. Yes, this looks good. Thanks for adding the tests! The docs might be a little bit too wordy, but I can work on that later. I try to use the space in README as efficiently as possible.

**Comment by vendruscolo** [comment]
Thank you! I'm sorry for being a bit too wordy on the main documentation, I had the same feeling but I still thought it was worth adding some more context about the possible use cases of this simple addition.

Thank you for your time! I apologize it took so long for completing this PR 😬
